Dockerfile

```diff
@@ -227,14 +252,14 @@ RUN apt-get update -qq && \
         build-essential \
         libffi-dev \
         libssl-dev \
-        python3.9 \
+        python3 \
```

Comment:
CLN still supports Python v3.10, while `bookworm` will auto-upgrade it to v3.11. As long as this is not causing any compatibility issues, it remains a minor concern. We will consider downgrading it to v3.10 only if any issues arise.